Assets/Scripts/Enemy/EnemyFactory.cs

```diff
@@ -56,6 +56,9 @@ public GameObject CreateEnemy(string enemyName, Vector3 position)
         // TODO: change default vector to dynamically adjust height of enemy spawn so they don't spawn under the ground.
         enemyTransform.position = position + new Vector3(0, 1, 0);
         
+        Weapon w = newEnemyObject.AddComponent<Weapon>();

```

Comment:
we really shouldnt do these 1 letter variable names

Reply:
Haha, I thought that to myself as I typed it out. Fixed.